### ISSUE TITLE
Extension is stealing focus (twice) outside of VsCode on `code .` on startup

### ISSUE THREAD

**Issue by tebeco** [Azure/vscode-bridge-to-kubernetes#47]
TLDR: Disabling the extensions stop the behavior
Spoiler: there's noise from `AKS` and `Kubernetes` extensions but the culprit seems to really be `bridge-to-kubernetes` (after disabling / enabling attempts)

**Describe the bug**
Pre-requisite:
* Windows
* Windows Terminal (aka `wt`)
* Use Windows Terminal as the default OS terminal

When the extension starts, like when using `code .` or disable / re-enable it it will apparently run 2 commands.
**These command are incorrectly started**, therefore it ALSO open 2 instances of `wt`.
These 2 instances of `wt` have some huge impact on the UX:
* they apparently steal focus and dont exactly know what/how to restore it
* VsCode goes out of focus
* VsCode can even go behind another app

**Mention the platform you are using**
* Windows
* Windows Terminal (aka `wt`)
* VsCode
* Use Windows Terminal as the default OS terminal

**To Reproduce**
0. See pre-requisite
1. Go to any folder from WT
2. type `code .`
3. wait, and you'll see the 2 intrusive popover show and close
4. you loose focus from the IDE that you just attempted to open

**Expected behavior**

**Screenshots**
Extensions should absolutly never open visible `WT` (or any other terminal) out of the IDE.

**Desktop (please complete the following information):**
 - OS: latest Win11
 - Browser: irrelevant
 - Version: v2.0.120230418

**Additional context**
I can't tell if that's new or not :'(
I don't exactly recall if that happen on re-install or later

Here's a first glance at it until i do manage to capture more
(Yes I know PID is useless ... it's just to show that it's definitly 2 different process ... for 2 different `wt` glitching)

https://github.com/Azure/vscode-bridge-to-kubernetes/assets/2266487/9c2a2d35-86ac-4c98-b565-c618237a08e1
![image](https://github.com/Azure/vscode-bridge-to-kubernetes/assets/2266487/a6647c33-9497-4527-918a-b4b11878ff68)
![image](https://github.com/Azure/vscode-bridge-to-kubernetes/assets/2266487/0b3858de-1d3c-4879-8c9a-c909d3e74e56)
![image](https://github.com/Azure/vscode-bridge-to-kubernetes/assets/2266487/361983f8-0973-48e5-a16c-8615af9e8e84)

Here's another run (THE PID WILL NOT MATCH)
This is the `ProcMon` dump of:
* `ProcessName contains kub`
* `Operation is process Start`
```
22:14:38.0544379	kubectl.exe	20368	Process Start		SUCCESS	Parent PID: 13192, Command line: C:\dev\tools\kubectl\kubectl.exe  config view -o json, Current directory: c:\dev\github\, Environment: 
22:14:38.0804003	kubectl.exe	10012	Process Start		SUCCESS	Parent PID: 49184, Command line: C:\dev\tools\kubectl\kubectl.exe  config view -o json, Current directory: c:\dev\github\, Environment: 
22:14:38.1258596	kubectl.exe	29996	Process Start		SUCCESS	Parent PID: 19252, Command line: C:\dev\tools\kubectl\kubectl.exe  config current-context, Current directory: c:\dev\github\, Environment: 
22:14:38.2675387	kubectl.exe	46628	Process Start		SUCCESS	Parent PID: 12932, Command line: C:\dev\tools\kubectl\kubectl.exe proxy --port=0, Current directory: c:\dev\github\, Environment: 
22:14:38.2935369	kubectl.exe	40812	Process Start		SUCCESS	Parent PID: 34356, Command line: C:\dev\tools\kubectl\kubectl.exe  api-versions, Current directory: c:\dev\github\, Environment: 
22:14:38.3060370	kubectl.exe	20880	Process Start		SUCCESS	Parent PID: 50944, Command line: C:\dev\tools\kubectl\kubectl.exe  config view -o json, Current directory: c:\dev\github\, Environment: 
22:14:38.3460138	kubectl.exe	9364	Process Start		SUCCESS	Parent PID: 23964, Command line: C:\dev\tools\kubectl\kubectl.exe  cluster-info, Current directory: c:\dev\github\, Environment: 
22:14:38.5258930	kubelogin.exe	27856	Process Start		SUCCESS	Parent PID: 40812, Command line: kubelogin get-token --environment AzurePublicCloud --server-id REDACTED --client-id REDACTED --tenant-id REDACTED --login devicecode, Current directory: c:\dev\github\, Environment: 
22:14:38.6086390	kubelogin.exe	47648	Process Start		SUCCESS	Parent PID: 9364, Command line: kubelogin get-token --environment AzurePublicCloud --server-id REDACTED --client-id REDACTED --tenant-id REDACTED --login devicecode, Current directory: c:\dev\github\, Environment: 
22:14:38.8228357	kubelogin.exe	36432	Process Start		SUCCESS	Parent PID: 46628, Command line: kubelogin get-token --environment AzurePublicCloud --server-id REDACTED --client-id REDACTED --tenant-id REDACTED --login devicecode, Current directory: c:\dev\github\, Environment: 
22:14:38.8240022	kubectl.exe	19824	Process Start		SUCCESS	Parent PID: 12932, Command line: "c:\Users\REACTED\AppData\Roaming\Code - Insiders\User\globalStorage\mindaro.mindaro\file-downloader-downloads\kubectl\win\kubectl.exe" version --short --client -o json, Current directory: C:\Users\REACTED\AppData\Local\Programs\Microsoft VS Code Insiders\, Environment: 
22:14:38.8364314	kubectl.exe	34372	Process Start		SUCCESS	Parent PID: 3756, Command line: C:\dev\tools\kubectl\kubectl.exe  cluster-info, Current directory: c:\dev\github\, Environment: 
22:14:38.9580671	kubelogin.exe	8596	Process Start		SUCCESS	Parent PID: 34372, Command line: kubelogin get-token --environment AzurePublicCloud --server-id REDACTED --client-id REDACTED --tenant-id REDACTED --login devicecode, Current directory: c:\dev\github\, Environment: 
22:14:40.2657799	kubectl.exe	41596	Process Start		SUCCESS	Parent PID: 39988, Command line: C:\dev\tools\kubectl\kubectl.exe  get crd -o name, Current directory: c:\dev\github\, Environment: 
22:14:40.4367394	kubectl.exe	49292	Process Start		SUCCESS	Parent PID: 12932, Command line: "c:\Users\REACTED\AppData\Roaming\Code - Insiders\User\globalStorage\mindaro.mindaro\file-downloader-downloads\bridge\kubectl\win\kubectl.exe" version --short --client -o json, Current directory: C:\Users\REACTED\AppData\Local\Programs\Microsoft VS Code Insiders\, Environment: 
22:14:40.6080335	kubelogin.exe	15748	Process Start		SUCCESS	Parent PID: 41596, Command line: kubelogin get-token --environment AzurePublicCloud --server-id REDACTED --client-id REDACTED --tenant-id REDACTED --login devicecode, Current directory: c:\dev\github\, Environment: 
22:14:40.6895192	kubectl.exe	51968	Process Start		SUCCESS	Parent PID: 12932, Command line: "c:\Users\REACTED\AppData\Roaming\Code - Insiders\User\globalStorage\mindaro.mindaro\file-downloader-downloads\kubectl\win\kubectl.exe" config view --minify -o json --kubeconfig C:\Users\REACTED\.kube\config, Current directory: C:\Users\REACTED\AppData\Local\Programs\Microsoft VS Code Insiders\, Environment: 
22:14:40.7061448	kubectl.exe	43212	Process Start		SUCCESS	Parent PID: 12932, Command line: "c:\Users\REACTED\AppData\Roaming\Code - Insiders\User\globalStorage\mindaro.mindaro\file-downloader-downloads\kubectl\win\kubectl.exe" config view --minify -o json --kubeconfig C:\Users\REACTED\.kube\config, Current directory: C:\Users\REACTED\AppData\Local\Programs\Microsoft VS Code Insiders\, Environment: 
22:14:40.8119608	kubectl.exe	40332	Process Start		SUCCESS	Parent PID: 12932, Command line: "c:\Users\REACTED\AppData\Roaming\Code - Insiders\User\globalStorage\mindaro.mindaro\file-downloader-downloads\kubectl\win\kubectl.exe" config view -o jsonpath={.clusters[*].cluster.server} --kubeconfig C:\Users\REACTED\.kube\config, Current directory: C:\Users\REACTED\AppData\Local\Programs\Microsoft VS Code Insiders\, Environment: 
22:14:41.2491242	kubelogin.exe	4244	Process Start		SUCCESS	Parent PID: 18224, Command line: "kubelogin" get-token --environment AzurePublicCloud --server-id REDACTED --client-id REDACTED --tenant-id REDACTED --login devicecode, Current directory: C:\Users\REACTED\AppData\Local\Programs\Microsoft VS Code Insiders\, Environment: 
22:14:42.2619781	kubectl.exe	17860	Process Start		SUCCESS	Parent PID: 3852, Command line: C:\dev\tools\kubectl\kubectl.exe  get crd -o jsonpath="{range .items[*].spec}[{.group}, {.names.plural}, {.names.singular}, {range .versions[?(.schema.openAPIV3Schema.properties)]}{.name}{\" \"}{end}]{end}", Current directory: c:\dev\github\, Environment: 
22:14:42.4532906	kubelogin.exe	15536	Process Start		SUCCESS	Parent PID: 17860, Command line: kubelogin get-token --environment AzurePublicCloud --server-id REDACTED --client-id REDACTED --tenant-id REDACTED --login devicecode, Current directory: c:\dev\github\, Environment: 
22:14:42.4887309	kubelogin.exe	28920	Process Start		SUCCESS	Parent PID: 18224, Command line: "kubelogin" get-token --environment AzurePublicCloud --server-id REDACTED --client-id REDACTED --tenant-id REDACTED --login devicecode, Current directory: C:\Users\REACTED\AppData\Local\Programs\Microsoft VS Code Insiders\, Environment: 
22:14:44.0147468	kubectl.exe	50508	Process Start		SUCCESS	Parent PID: 47852, Command line: C:\dev\tools\kubectl\kubectl.exe  get crd secretproviderclasses.secrets-store.csi.x-k8s.io  -o json, Current directory: c:\dev\github\, Environment: 
22:14:44.1445215	kubelogin.exe	43412	Process Start		SUCCESS	Parent PID: 50508, Command line: kubelogin get-token --environment AzurePublicCloud --server-id REDACTED --client-id REDACTED --tenant-id REDACTED --login devicecode, Current directory: c:\dev\github\, Environment: 
22:14:45.2127570	kubectl.exe	38696	Process Start		SUCCESS	Parent PID: 50208, Command line: C:\dev\tools\kubectl\kubectl.exe  get crd secretproviderclasspodstatuses.secrets-store.csi.x-k8s.io  -o json, Current directory: c:\dev\github\, Environment: 
22:14:45.3967931	kubelogin.exe	12892	Process Start		SUCCESS	Parent PID: 38696, Command line: kubelogin get-token --environment AzurePublicCloud --server-id REDACTED --client-id REDACTED --tenant-id REDACTED --login devicecode, Current directory: c:\dev\github\, Environment: 
22:14:45.6639581	kubectl.exe	41776	Process Start		SUCCESS	Parent PID: 12932, Command line: "c:\Users\REACTED\AppData\Roaming\Code - Insiders\User\globalStorage\mindaro.mindaro\file-downloader-downloads\kubectl\win\kubectl.exe" config view --minify -o json --kubeconfig C:\Users\REACTED\.kube\config, Current directory: C:\Users\REACTED\AppData\Local\Programs\Microsoft VS Code Insiders\, Environment: 
22:14:46.1037211	kubectl.exe	48908	Process Start		SUCCESS	Parent PID: 12932, Command line: "c:\Users\REACTED\AppData\Roaming\Code - Insiders\User\globalStorage\mindaro.mindaro\file-downloader-downloads\kubectl\win\kubectl.exe" get ingress -n default -o json --kubeconfig C:\Users\REACTED\.kube\config, Current directory: C:\Users\REACTED\AppData\Local\Programs\Microsoft VS Code Insiders\, Environment: 
22:14:46.1129540	kubectl.exe	44932	Process Start		SUCCESS	Parent PID: 12932, Command line: "c:\Users\REACTED\AppData\Roaming\Code - Insiders\User\globalStorage\mindaro.mindaro\file-downloader-downloads\kubectl\win\kubectl.exe" get services -n default -o json --kubeconfig C:\Users\REACTED\.kube\config, Current directory: C:\Users\REACTED\AppData\Local\Programs\Microsoft VS Code Insiders\, Environment: 
22:14:46.3016133	kubelogin.exe	48688	Process Start		SUCCESS	Parent PID: 48908, Command line: kubelogin get-token --environment AzurePublicCloud --server-id REDACTED --client-id REDACTED --tenant-id REDACTED --login devicecode, Current directory: C:\Users\REACTED\AppData\Local\Programs\Microsoft VS Code Insiders\, Environment: 
22:14:46.3115869	kubelogin.exe	21088	Process Start		SUCCESS	Parent PID: 44932, Command line: kubelogin get-token --environment AzurePublicCloud --server-id REDACTED --client-id REDACTED --tenant-id REDACTED --login devicecode, Current directory: C:\Users\REACTED\AppData\Local\Programs\Microsoft VS Code Insiders\, Environment: 
22:14:46.5022642	kubectl.exe	43132	Process Start		SUCCESS	Parent PID: 50996, Command line: C:\dev\tools\kubectl\kubectl.exe  get crd volumesnapshotclasses.snapshot.storage.k8s.io  -o json, Current directory: c:\dev\github\, Environment: 
22:14:46.7976376	kubelogin.exe	48140	Process Start		SUCCESS	Parent PID: 43132, Command line: kubelogin get-token --environment AzurePublicCloud --server-id REDACTED --client-id REDACTED --tenant-id REDACTED --login devicecode, Current directory: c:\dev\github\, Environment: 
22:14:47.9930616	kubectl.exe	47560	Process Start		SUCCESS	Parent PID: 37188, Command line: C:\dev\tools\kubectl\kubectl.exe  get crd volumesnapshotcontents.snapshot.storage.k8s.io  -o json, Current directory: c:\dev\github\, Environment: 
22:14:48.2091339	kubelogin.exe	32212	Process Start		SUCCESS	Parent PID: 47560, Command line: kubelogin get-token --environment AzurePublicCloud --server-id REDACTED --client-id REDACTED --tenant-id REDACTED --login devicecode, Current directory: c:\dev\github\, Environment: 
22:14:49.4996502	kubectl.exe	30204	Process Start		SUCCESS	Parent PID: 45776, Command line: C:\dev\tools\kubectl\kubectl.exe  get crd volumesnapshots.snapshot.storage.k8s.io  -o json, Current directory: c:\dev\github\, Environment: 
22:14:49.5845756	kubelogin.exe	46920	Process Start		SUCCESS	Parent PID: 30204, Command line: kubelogin get-token --environment AzurePublicCloud --server-id REDACTED --client-id REDACTED --tenant-id REDACTED --login devicecode, Current directory: c:\dev\github\, Environment: 
```

**Comment by tebeco** [comment]
unsure if issue are activly tracked
